resources/views/components/inputRead.blade.php

```diff
@@ -0,0 +1,11 @@
+<div class="form-group">
+    <label for="{{ $name }}">{{ $displayedName }}</label>
+    <input type="{{ $type }}"
+           class="form-control {{ isset($errorMessaage) ? 'is-invalid' : '' }}"
+           name="{{ $name }}"
+           value="{{ old($name, $value) }}"
+           readonly>
```

Comment:
You could set readonly parameter by adding property to existing input component. Otherwise good job for creating your own.